codegen/src/Azure.Iot.Operations.ProtocolCompiler/T4/communication/rust/Service/t4/RustService.tt

```diff
@@ -8,7 +8,10 @@ mod <#=Path.GetFileNameWithoutExtension(module)#>;
 
 pub use azure_iot_operations_protocol::common::aio_protocol_error::AIOProtocolError;
 
-pub use super::common_types::common_options::{CommandOptions, TelemetryOptions};
+pub use super::common_types::options::{
+    CommandExecutorOptions, CommandInvokerOptions,
```

Comment:
Should this only import the ones that will be used? (since we know whether we're only generating service or client side?)